usr/share/rear/restore/NFS_SERVER/default/400_restore_with_nfs_server.sh

```diff
@@ -0,0 +1,19 @@
+# 400_restore_with_nfs_server.sh
+
+local check_file=$TARGET_FS_ROOT/$NFS_SERVER_RESTORE_FINISHED_FILE
+
+Print "Mount the nfs share: 'mount -t nfs -o nfsvers=4 <ip>:/ <destination>' and restore all files to mounted destination."
```

Comment:
```suggestion
LogPrint "Mount the nfs share: 'mount -t nfs -o nfsvers=4 <ip>:/ <destination>' and restore all files to mounted destination."
```

Reply:
`nfsvers=4.2`? (Because of extended attribute support.) I have not tried that yet.

Reply:
Without version specification, the client takes the highest available version. 
In the LogPrint message I have removed the version. Versions smaller than 4 are not allowed.